blockonomics-woocommerce.php

```diff
@@ -410,10 +410,52 @@ function show_basic() {
         </div>
     <?php
     }
+
+    function updateTable($order, $email=false)
+    {
+        global $wpdb;
+        $order_id = $order->get_id();
+        $results = array();
+        $table_name = $wpdb->prefix .'blockonomics_payments'; 
+        $query = $wpdb->prepare("SELECT expected_fiat,paid_fiat,currency FROM ". $table_name." WHERE order_id = " . $order_id);
+        $results = $wpdb->get_results($query,ARRAY_A);
+        if(!is_array($results) || empty($results)) {
+          return;
+        }
+        $order_data = $results[0];
+        $paid_fiat = 0;
+        foreach ($results as $row) {
+        $paid_fiat = $paid_fiat + (float)$row['paid_fiat'];
+        }
+        $expected_fiat = (float)$order->get_total();
+        if ($paid_fiat >= $expected_fiat) {
+          return;
+        }
+        $currencyCode = $order_data['currency'];
+        $formatter = NumberFormatter::create(NULL, NumberFormatter::CURRENCY);
+        $formatter->setTextAttribute(NumberFormatter::CURRENCY_CODE, $currencyCode);
+        $remaining_fiat = $expected_fiat - $paid_fiat;
+        $formatted_amount_expected_fiat = $formatter->formatCurrency($paid_fiat, $currencyCode);
+        $formatted_amount_remaining_fiat = $formatter->formatCurrency($remaining_fiat, $currencyCode);
+        
+        if ($email) { 
```

Comment:
The changes for the email will not work if the user changes the email color settings or template. Let's use the same `<table class="woocommerce-table` for both the email and theme to keep things simple for now. This will display as a basic table:
![Email](https://user-images.githubusercontent.com/39666372/235468506-aaa5f77f-5dce-4649-8dd3-d610c7557db1.png)